html-forms/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+<h1>Coding Form!</h1>
+<form>
+  <label>
+    Email
+    <div>
+      <input type="text" name="email" maxlength="30">
+    </div>
+  </label>
+  Why is coding cool?
+  <div>
+  <select name="why">
+    <option value="select">Select an option</option>
+    <option value="interesting">Because it's interesting</option>
+    <option value="fun">Because it's fun</option>
+    <option value="powerful">Because it's powerful</option>
+  </select>
+  </div>
+  Favorite Language (choose one)
```

Comment:
You should put this into a text element such as `p` rather than having free-floating text content in the document.